entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo $DB_CONFIG > /etc/sns/db.json
```

Comment:
Should we assume that the DB_PATH hasn't been overridden with this? Otherwise I can make it so it echos the config into the path defined by DB_PATH.

I'll fix the overwriting of existing config now, just realised that, thanks! 😄